src/features/campaigns/components/CampaignActionButtons.tsx

```diff
@@ -134,17 +84,24 @@ const CampaignActionButtons: React.FunctionComponent<
             {
               icon: <HeadsetMic />,
               label: messages.linkGroup.createCallAssignment(),
-              onClick: handleCreateCallAssignment,
+              onClick: () =>
+                createCallAssignment({
+                  title: messages.form.createCallAssignment.newCallAssignment(),
+                }),
```

Comment:
The rerouting that currently happens in a middleware could probably be solved like this:

```ts
onClick: async () => {
  const callAssignment = await createCallAssignment({
    title: messages.form.createCallAssignment.newCallAssignment(),
  });
  router.push(`/organize/${callAssignment.organization?.id}/projects/${campId}/callassignments/${callAssignment.id}`);
},
```